File: firstApp/app/src/main/java/com/example/firstapp/MainActivity.java
package com.example.firstapp;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Spinner;
import android.widget.TextView;

import java.util.List;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void onClickFindBeeer(View view) {
        TextView brands = findViewById(R.id.textViewBeer);
        Spinner colorSpinner = findViewById(R.id.spinnerColor);
        String beerType = String.valueOf(colorSpinner.getSelectedItem());
        List<String> brandList;
        brandList = new BeerExpert().getBrands(beerType);
        StringBuilder brandsFormatted = new StringBuilder();
        for (String brand: brandList)
            brandsFormatted.append(brand).append('\n');
        brands.setText(brandsFormatted);
    }
}

File: horoscope/app/src/main/java/com/example/horoscope/MainActivity.kt
package com.example.horoscope

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.CalendarView
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.view.*

class MainActivity : AppCompatActivity() {
    lateinit var calendarV: calendarView;
    lateinit var textV: ;

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        calendarV.calendarView = R.id.calendarView
        textV.textView = R.id.textView

        calendarV.setOnDateChangeListener {
            textV.text = "asdfasdf"

        }
    }
}
